packages/ui/src/ui/pages/navigation/content/MapNode/MapNodesTableIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icon from '../../../../components/Icon/Icon';
+import {getIconNameForType} from '../../../../utils/navigation/path-editor';
+
+const TRASH_PATH = '//tmp/trash';
+
+export function isTrashNode(item: any) {
```

Comment:
It is always bad idea to use `any` this way. Required type might be easily inferred from implementation:
```ts
export function isTrashNode(item: {path: string}) {
    ...
}
```